p5js/02_KNNImageClassification/sketch.js

```diff
@@ -16,6 +16,18 @@ function setup() {
 }
 
 function createButtons() {
+  // Save and Load buttons
+  save = select('#save');
+  save.mousePressed(() => {
```

Comment:
Should probably remove `=>` (for now at least)

Reply:
Oh, I missed that. Updated in the latest commit